src/content/docs/apis/nerdgraph/get-started/nerdgraph-explorer.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: Use the NerdGraph explorer
+tags:
+  - APIs
+  - NerdGraph
+  - Get started
+metaDescription: How to use the New Relic NerdGraph API explorer. 
+redirects:
+---
+
+import createAccount from 'images/create-account.png'
+
+import graphqlDocumentation from 'images/graphql-documentation.png'
+
+import mutationExample from 'images/mutation-example.png'
+
+import toolsMenu from 'images/tools-menu.png'
+
+NerdGraph is our main API for querying data and making configurations to New Relic features. One popular way to use NerdGraph and make API requests is with our NerdGraph explorer. 
+
+## Find the explorer [#find]
+
+To find the explorer: 
+
+* If your New Relic account uses an [EU data center](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers), go to [api.eu.newrelic.com/graphiql](https://api.eu.newrelic.com/graphiql).
+* Otherwise use [api.newrelic.com/graphiql](https://api.newrelic.com/graphiql).
+
+## What is the NerdGraph explorer? [#overview]
+
+Our NerdGraph API is built using GraphQL, a third-party query language. Our NerdGraph explorer is built using GraphiQL, which is an open source graphical interface for forming and making GraphQL requests. 
+
+You can use the NerdGraph explorer to explore our data schema, to read built-in object definitions, and to build and execute queries.
+
+To watch a short video showing how to use the explorer, see [Introduction to NerdGraph](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph).
+
+## Before you begin 
+
+Before you do the lessons here, be sure you've reviewed [Introduction to NerdGraph](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph). 
+
+Go to [the explorer](#find) and log in using your New Relic user ID and password.
+
+Make sure you have a valid [New Relic user key](/docs/apis/get-started/intro-apis/types-new-relic-api-keys#user-api-key). You can create one from the NerdGraph explorer or view and create them in the [API keys UI](https://one.newrelic.com/launcher/api-keys-ui.api-keys-launcher).
+
+<img
+  title="Create a user key"
+  alt="Create a user key"
+  src={createAccount}
+/>
+
+Next we'll walk you through some steps that will help you learn to use the explorer. 
+
+## Step 1. Build a query to retrieve your name [#build-query]
+
+Time for your first NerdGraph query. Search for your name in the New Relic database:
```

Comment:
@zuluecho9, non-native speakers may not get this fragment. Maybe just modify to this: Try your first NerdGraph query."